src/Rush/MessageSender.cs

```diff
@@ -39,7 +39,7 @@ public Task SendAsync(T message, CancellationToken cancellationToken)
 			{
 				await operationalChannel.SendAsync(message, cancellationToken).ConfigureAwait(false);
 			}
-			catch (Exception ex)
+			catch (Exception ex) when (ex.GetType() != typeof(OperationCanceledException))
```

Comment:
can't decide what's better...

`catch (OperationCanceledException) { }`

or using the inverted filter.